src/rpc/handlers/NFTOffers.cpp

```diff
@@ -100,13 +97,13 @@ enumerateNFTOffers(
         cursor,
         0,
         lgrInfo.seq,
-        limit,
+        reserve,
         {},
         context.yield,
         [&offers](ripple::SLE const& offer) {
             if (offer.getType() == ripple::ltNFTOKEN_OFFER)
             {
-                offers.emplace_back(offer);
+                offers.push_back(offer);
```

Comment:
can we actually move this offer object?

Reply:
not as it was (`const ref`). now we can tho :)